funnel/models/space.py

```diff
@@ -65,6 +87,7 @@ class ProposalSpace(BaseScopedNameMixin, db.Model):
     parent_space_id = db.Column(None, db.ForeignKey('proposal_space.id', ondelete='SET NULL'), nullable=True)
     parent_space = db.relationship('ProposalSpace', remote_side='ProposalSpace.id', backref='subspaces')
     inherit_sections = db.Column(db.Boolean, default=True, nullable=False)
+    labels = db.Column(JsonDict, nullable=False, server_default=json.dumps(labels_default()))
```

Comment:
Avoid `server_default` if possible (purely for reason of convention). Let this be a client default. You also don't need `json.dumps` in this case, the column will massage a dictionary into JSON automatically.